<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="iterations_label">
          <property name="text">
           <string>Iterations</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="iterations"/>
        </item>
        <item>
         <widget class="QLabel" name="nx_label">
          <property name="text">
           <string>nx</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nx"/>
        </item>
        <item>
         <widget class="QLabel" name="ny_label">
          <property name="text">
           <string>ny</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ny"/>
        </item>
        <item>
         <widget class="QLabel" name="cx_label">
          <property name="text">
           <string>cx</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="cx">
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="cy_label">
          <property name="text">
           <string>cy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="cy">
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="interiorvalue_label">
          <property name="text">
           <string>Interior value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="interiorvalue"/>
        </item>
        <item>
         <widget class="QLabel" name="jobname_label">
          <property name="text">
           <string>Jobname</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="jobname"/>
        </item>
        <item>
         <widget class="QPushButton" name="submitbutton">
          <property name="text">
           <string>Submit</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="imagepix">
        <property name="text">
         <string>                                                       Matplotlib image                                                                          </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionfileopen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuedit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuedit"/>
  </widget>
  <action name="actionfileopen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Sa&amp;ve as ...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPast">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
